var/spack/repos/builtin/packages/acts-dfelibs/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class ActsDfelibs(CMakePackage):
```

Comment:
```suggestion
class Dfelibs(CMakePackage):
```

We typically avoid naming packages differently for forks than for their upstream. I understand that upstream is unmaintained, but this avoids that someone else creates another `dfelibs` package independently of this one.

If at some point someone wants to add upstream (and cannot use the acts fork), or another fork, and if there really are conflicts between the patch sets that prevent merging into acts as a de facto maintained version, then we can always figure out something then.